### PR TITLE
Add simple Catalyst test using content generated with catalyst.pl MyApp

### PR DESCRIPTION
**Pull request by nicomen**

### CODE REVIEW

File: t/catalyst/catalyst.t [hunk 1]
```diff
@@ -0,0 +1,14 @@
+use Test::More;
+
+use Test::Mojo::WithRoles 'PSGI';
+
+use FindBin;
+use lib "$FindBin::Bin/MyApp/lib";
+
+plan( skip_all => 'Tests need Catalyst installed to run' ) unless eval { require 'Catalyst.pm'; };
+
+my $t1 = Test::Mojo::WithRoles->new('./MyApp/myapp.psgi');
```
Comment (jberger) [inline t/catalyst/catalyst.t]: Is this portable?

**Comment by jberger** [comment]
also, does all the boilerplate need to be in there? seems a little untidy

File: t/catalyst/catalyst.t (hunk 1 above)
Comment (nicomen) [inline t/catalyst/catalyst.t]: The paths you mean? I guess not.

**Comment by nicomen** [comment]
I wanted it to be as close to what came out of catalyst.pl as possible. If you want I can of course remove all the cruft.

**Comment by nicomen** [comment]
Did you find out what is wrong?

**Comment by jberger** [comment]
Thanks. For the record the problem was that Catalyst uses the deferred PSGI response pattern which Mojolicious::Plugin::MountPSGI did not support. That has been fixed in 0.05 release of that module. I have also released a new version of this module which requires that version and which provides these tests. 

Cheers!

**Pull request merged**